images/go-grpc-greeter-server/cloudbuild.yaml

```diff
@@ -0,0 +1,24 @@
+timeout: 1200s
+options:
+  substitution_option: ALLOW_LOOSE
+  # job builds a multi-arch docker image for amd64,arm,arm64 and s390x.
+  machineType: N1_HIGHCPU_8
+steps:
+  - name: gcr.io/k8s-testimages/gcb-docker-gcloud:v20201130-750d12f
```

Comment:
There's a brand new version for this: v20210622-762366a

Let's use it

Reply:
changed